scripts/merge-gisaid-owid.py

```diff
@@ -389,6 +390,13 @@ def main(args_list=None):
         # precalculate cases per mil and percent sequenced for all rows
         weekly_df = calculate_cols(weekly_df)
         
+        # cut off weekly timeseries at most recent completed week of reporting
+        latest_week_numdays = (max_gisaid_date - weekly_df['gisaid_collect_weekstartdate'].max()).days 
+        print(f'Latest GISAID submission date: {max_gisaid_date}, latest week starting Monday: {weekly_df["gisaid_collect_weekstartdate"].max()}')
+        if latest_week_numdays < 6:
+            print('Dropping latest week because of incomplete data')
+            weekly_df = weekly_df.loc[(weekly_df.gisaid_collect_weekstartdate < weekly_df['gisaid_collect_weekstartdate'].max())]
```

Comment:
one tiny thing - for readability i'd pull `weekly_df['gisaid_collect_weekstartdate'].max()` into a variable since you're using it a few times, but otherwise looks good to merge!

Reply:
Thanks, great point. Will do in the next PR